packages/ragbits-document-search/src/ragbits/document_search/retrieval/rerankers/litellm.py

```diff
@@ -0,0 +1,51 @@
+from typing import List
+
+import litellm
+
+from ragbits.document_search.documents.element import Element, TextElement
+from ragbits.document_search.retrieval.rerankers.base import Reranker
+
+
+class LiteLLMReranker(Reranker):
```

Comment:
you should add it here: https://github.com/deepsense-ai/ragbits/blob/main/packages/ragbits-document-search/src/ragbits/document_search/retrieval/rerankers/__init__.py#L9 for it to work with config like this:
```
"reranker": {
        "type": "LiteLLMReranker",
        "config": {"model": "cohere/rerank-english-v3.0"},
    }
```